README.md

```diff
@@ -50,30 +50,32 @@ For Microverse based in these [requirements](https://www.theodinproject.com/cour
 
 # Features
 
-[![Framework][badge-framework]][framework-url]
-![javascript][]
+[![][javascript]][javascript-url] 
+[![][react]][react-url] 
 ![eslint][]
 
-- Basic VanillaJS app with a single script or few commands
+- Next.js
+- React
+- Next.js-Create-App
 - JS modules and ES6+ available for most browsers
-- connected an API using `async/await`
+- Connected an API using `async/await`
 - Sets `stickler` on the repo
 - Sets `eslint` rules
 - Tests made with [`ESLint`](https://eslint.org/)
```

Comment:
This one I think is not valid, as no tests were added

Reply:
I'll add tests